### DISCUSSION TITLE
v1.4-alpha.1

### DISCUSSION THREAD

**Discussion by arekf** [yattee/yattee#75] [Announcements]
## What's new

* **New player** built with [MPV](https://mpv.io/)
   * **4K** playback support
   * Much better at keeping playing high quality adaptive streams
* **New player controls**
* **Player gestures**: tap twice on the left/center/right of player to go back 10s/play-pause/go forward 10s
* Dislikes counter from [Return YouTube Dislike](https://returnyoutubedislike.com/) - disabled by default, to enable in Player settings
* Reorganized browser toolbars
  * Player always stays in same bottom-most place
  * Better translucency effect
* Fixed parsing video date in subscriptions
* Other minor changes

## What's missing (comparing to v1.3)
* Picture-in-Picture and AirPlay
* Subtitles
* Playback rate change
* **tvOS build is not yet available** - coming in the near future

## Known issues

* Player queue may not work properly
* Video may close prematurely
* Multiple seeks may cause crash
* Some FPS may be dropped when presenting controls
* Scrolling may stuck on iOS 14 during playback

**Full Changelog**: https://github.com/yattee/yattee/compare/main...v1.4-alpha.1

<hr /><em>This discussion was created from the release <a href='https://github.com/yattee/yattee/releases/tag/v1.4-alpha.1'>v1.4-alpha.1</a>.</em>

**Comment by bachig26** [comment]
### Pros:
1. With the **New player, the playback is smooth and works well** than the previous [v1.3.2 version](https://github.com/yattee/yattee/releases/tag/v1.3.2) player. **Video Quality stays as selected for the entirety** unlike the previous HLS based which used to differ quality from low to high automatically like a rollercoaster during the video playback.
2. plays the video instantly after changing the streaming source with a top-left option in the video source selector.
3. player gesture: double-tap works well.
4. Dislike counter works well.

### Cons:
**1. With the New player, the new controls are too big comparatively. [Image below!!!]
2. fast seeking through the video makes the video frame pause and only the audio is played.
3. I believe app size increased significantly with [MPV](https://mpv.io/) implementation.**

_!!![Request] Need a change in the player interface {more refined and clean}, smaller sized controls + possible removal of the color in the seek bar.🤞_

Also looking forward to have the PiP and subtitles option in the public release.❤

### Screenshot:
![IMG_6470](https://user-images.githubusercontent.com/96393348/159210264-03b3d614-2279-4c47-a73e-25e9daf6e2c8.jpg)
![IMG_6471](https://user-images.githubusercontent.com/96393348/159210270-1f69a294-13ad-4e9d-8da5-cbba5cdc2254.PNG)
